PubCloud enhancements (#272)
uyuni-project/uyuni-docs: modules/quickstart-public-cloud/pages/clients.adoc

When you have your {productname} Server set up, you are ready to start registering clients.

> {productname} supports management of clients using either the Salt stack or the traditional stack (inherited from Spacewalk). Most new features and enhancements are only available for Salt-managed clients, which makes this stack the preferred one. For more details, see xref:client-configuration:supported-features.adoc[]

Reviewer: ```suggestion
You can use {productname} to manage clients using either the Salt stack or the traditional stack (inherited from Spacewalk). 
Most new features and enhancements are only available for Salt-managed clients, which makes this stack the preferred one. 
For more details, see xref:client-configuration:supported-features.adoc[]
```
